src/__tests__/visualTesting/custom.css

```diff
@@ -0,0 +1,3 @@
+nav {
+  position: static !important;
```

Comment:
Could you explain why is this necessary?

Reply:
You have a sharp eye! Now it is not necessary since I've deleted separate test for footer section. I will delete custom css, thank you.
Just for clarification for others and sharing experience: sticky `nav` section covered footer on screenshots. Changing this css rule fixed this issue.

![footer-Mobile-Chrome-win32](https://github.com/rolling-scopes/site/assets/46446962/2f3dbf70-28bd-4b41-90cd-647f53858f31)